From d7a6b79ef1f35516f665f352b60268e8382c3665 Mon Sep 17 00:00:00 2001
From: Xiao-Long Chen <chenxiaolong@cxl.epac.to>
Date: Wed, 16 Apr 2014 19:34:55 -0400
Subject: [PATCH 3/4] Fix nosetests path

---
 make-test | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/make-test b/make-test
index 2a48ab1..849b88b 100755
--- a/make-test
+++ b/make-test
@@ -12,7 +12,7 @@ from subprocess import call
 
 versions = ('2.4', '2.5', '2.6', '2.7')
 python = '/usr/bin/python'
-nose = '/usr/bin/nosetests'
+nose = '/usr/bin/nosetests2'
 ran = []
 fail = []
 
-- 
2.2.1

