From a362e1664cf6b7b1dc531b4d082e22e79301ad3f Mon Sep 17 00:00:00 2001
From: Xiao-Long Chen <chenxiaolong@cxl.epac.to>
Date: Fri, 2 Jan 2015 01:03:15 -0500
Subject: [PATCH 4/4] Fix import for match_hostname

---
 ipalib/plugins/otptoken.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ipalib/plugins/otptoken.py b/ipalib/plugins/otptoken.py
index f085085..559e518 100644
--- a/ipalib/plugins/otptoken.py
+++ b/ipalib/plugins/otptoken.py
@@ -25,7 +25,7 @@ from ipalib.errors import PasswordMismatch, ConversionError, LastMemberError, No
 from ipalib.request import context
 from ipalib.frontend import Local
 
-from backports.ssl_match_hostname import match_hostname
+from ssl import match_hostname
 import base64
 import uuid
 import urllib
-- 
2.2.1

